examples/ChatQnA/deploy/gaudi.md

```diff
@@ -139,20 +142,20 @@ docker build --no-cache -t opea/reranking-tei:latest --build-arg https_proxy=$ht
 
 Build vLLM docker image with hpu support
 ```
-docker build --no-cache -t opea/llm-vllm-hpu:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_hpu .
+docker build --no-cache -t opea/llm-vllm-hpu:${TAG} --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_hpu .

```

Comment:
I'm getting:
```
ls: cannot access 'comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_hpu': No such file or directory
```
and the only Dockerfile I see in that location is:
`comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_gpu`

Reply:
I think that means, we need:
```
docker build --no-cache -t opea/llm-vllm-gpu:${TAG} --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_gpu .
```
or something like that in the README.

Reply:
Need to build for Gaudi/HPU, so using this step instead:

bash ./build_docker_vllm.sh hpu